src/app/pages/FastBtcPage/contexts/deposit-context.ts

```diff
@@ -0,0 +1,68 @@
+import { createContext, Dispatch, SetStateAction } from 'react';
+import { Nullable } from 'types';
+
+export enum DepositStep {
+  MAIN,
+  AMOUNT,
+  ADDRESS,
+  REVIEW,
+  CONFIRM,
+  PROCESSING,
+  COMPLETED,
+}
+
+export type DepositContextStateType = {
+  step: DepositStep;
+  ready: boolean;
+  address: string;
+  addressLoading: boolean;
+  addressError: Nullable<string>;
+  depositTx: Nullable<TxData>;
+  transferTx: Nullable<TxData>;
+  limits: DepositLimits;
+};
+
+type DepositLimits = {
+  min: number;
+  max: number;
+  loading: boolean;
+};
+
+export type TxData = {
+  txHash: string;
+  value: number;
+  status: TxStatus;
+};
+
+type TxStatus = string;
```

Comment:
I think that `TxStatus` should be an `enum`. If you think that's not the case, then you can delete this type definition and use `string` instead.

Reply:
Changed it to `'pending' | 'completed' | string` until backend will came back with full list of available states.